core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -555,6 +557,30 @@ public final void setTlsEnabled(final boolean enabled) {
         this.tlsEnabled = enabled;
     }
 
+    /**
+     * Get the host override for TLS SNI Server Name set by {@link #setSniServerName(String)}.
+     *
+     * @return  the sniServerName
+     */
+    public final String getSniServerName() {
+        return sniServerName;
+    }
+
+    /**
+     * Explicitly override the host to use for the TLS SNI server name.
+     *
+     * If neither the {@link ClientConfigProperties#setHost(String) host} or SNI server name is explicitly
+     * overridden, the host specified in {@link ClientConfigProperties#getHost()} will be used, with SNI performed implicitly
```

Comment:
The *host* is used if the *SniServerName* hasn't been set explicitly, isn't it?

Reply:
How I know this is the default behavior of ` io.vertx.proton.ProtonClient` :  if SNI Server name is not specified `host `name is used.

Reply:
https://vertx.io/docs/vertx-core/java/#ssl